packages/p2p-v2/src/components/AdvertiserName/AdvertiserNameToggle.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useEffect, useState, memo } from 'react';
+import { useAdvertiserStats } from '../../hooks';
+import { ToggleSwitch } from '../ToggleSwitch';
+import { useAdvertiserUpdate } from '@deriv/api';
```

Comment:
This is now inside the entity. You may use this `p2p.advertiser.useAdvertiserUpdate`.